tabs/pid_tuning.html

```diff
@@ -66,13 +66,14 @@
                     <div class="gui_box grey topspacer" style="margin-top: 0px;">
                         <table class="pid_titlebar">
                             <tr>
-                                <th class="name" i18n="pidTuningName"></th>
-                                <th class="proportional" i18n="pidTuningProportional"></th>
-                                <th class="integral" i18n="pidTuningIntegral"></th>
-                                <th class="derivative" i18n="pidTuningDerivative"></th>
-                                <th class="rc_rate" i18n="pidTuningRcRate"></th>
-                                <th class="rate" i18n="pidTuningRate"></th>
-                                <th class="rc_expo" i18n="pidTuningRcExpo"></th>
+                                <th class="name" i18n="pidTuningName" style="width: 12.5%;"></th>
```

Comment:
@borisbstyle: Can you please revert this? No point totally messing up the html and only creating more work for @skaman82